modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -105,7 +105,6 @@ def veteran_meta
         @evss_claim[:veteran][:currentlyVAEmployee] = @data.dig(:veteranIdentification, :currentVaEmployee)
         email_address = @data.dig(:veteranIdentification, :emailAddress, :email)
         @evss_claim[:veteran][:emailAddress] = email_address unless email_address.nil?
-        @evss_claim[:veteran][:fileNumber] = @file_number
```

Comment:
since we got time til merge, let's clean up the initialize and all the service calls so we're completely removed from this. 

https://github.com/department-of-veterans-affairs/vets-api/blob/0469d7b22e8e04bdbda2e6ede6b189e93c6804b2/modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb#L6
https://github.com/department-of-veterans-affairs/vets-api/blob/0469d7b22e8e04bdbda2e6ede6b189e93c6804b2/modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb#L10
https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/sidekiq/claims_api/service_base.rb#L179-L180
https://github.com/department-of-veterans-affairs/vets-api/blob/0469d7b22e8e04bdbda2e6ede6b189e93c6804b2/modules/claims_api/app/services/claims_api/disability_compensation/docker_container_service.rb#L46
https://github.com/department-of-veterans-affairs/vets-api/blob/0469d7b22e8e04bdbda2e6ede6b189e93c6804b2/modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_docker_container_upload.rb#L21